app/controllers/components/course/materials_component.rb

```diff
@@ -0,0 +1,15 @@
+class Course::MaterialsComponent < SimpleDelegator
+  include Course::ControllerComponentHost::Component
+
+  def sidebar_items
+    [
+      {
+        key: :materials,
+        title: I18n.t('course.material.sidebar_title'),
+        weight: 4,
+        path: course_material_folder_path(current_course, current_course.root_folder),
```

Comment:
If changes are in the schema, we will use migration scripts.

But this is not related to schema. 

I understand, I mean using models in rake tasks. not migration.